### PR TITLE
Bug 1802230: Fix missing services in VM details page

### PR DESCRIPTION
**Pull request by yaacov**
Fix missing services in VM details page

vmi services selector is broken, this PR fixes the services selector.

Screenshot:
OKD

### CODE REVIEW

**Comment by yaacov** [comment]
@rhrazdil @suomiy @glekner please review

File: frontend/packages/kubevirt-plugin/src/selectors/service/selectors.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ const getServiceSelectors = (service: ServiceKind) =>
   service && service.spec && service.spec.selector ? service.spec.selector : {};
 
 export const getServicesForVmi = (services: ServiceKind[], vmi: VMIKind): ServiceKind[] => {
-  const vmLabels = getVmiTemplateLabels(vmi);
+  const vmLabels = getVMILabels(vmi);
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/selectors/service/selectors.ts]: could we rather use `getLabels` from `console-shared`?

File: frontend/packages/kubevirt-plugin/src/selectors/vmi/basic.ts [hunk 2]
```diff
@@ -28,8 +28,8 @@ export const getVMIConditionsByType = (
   return (conditions || []).filter((cond) => cond.type === condType);
 };
 
-export const getVmiTemplateLabels = (vmi: VMIKind): { [key: string]: string } =>
-  (_.get(vmi, 'vmi.spec.template.metadata') && vmi.spec.template.metadata.labels) || {};
+export const getVMILabels = (vmi: VMIKind): { [key: string]: string } =>
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/selectors/vmi/basic.ts]: and delete this one?

File: frontend/packages/kubevirt-plugin/src/selectors/service/selectors.ts (hunk 1 above)
Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/selectors/service/selectors.ts]: Nice :exclamation: 
done

File: frontend/packages/kubevirt-plugin/src/selectors/service/selectors.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ const getServiceSelectors = (service: ServiceKind) =>
   service && service.spec && service.spec.selector ? service.spec.selector : {};
 
 export const getServicesForVmi = (services: ServiceKind[], vmi: VMIKind): ServiceKind[] => {
-  const vmLabels = getVmiTemplateLabels(vmi);
+  const vmLabels = getLabels(vmi);
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/selectors/service/selectors.ts]:
```suggestion
  const vmLabels = getLabels(vmi, {});
```
to prevent NPEs

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/selectors/service/selectors.ts]: Thanks !

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/selectors/service/selectors.ts]: done

**Pull request merged**